common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -1,27 +1,47 @@
 """
 VerticalBlock - an XBlock which renders its children in a column.
 """
-import logging
 from copy import copy
+import logging
 
 from lxml import etree
+from opaque_keys.edx.keys import UsageKey
+import six
 from xblock.core import XBlock
+from xblock.exceptions import JsonHandlerError
 from xblock.fragment import Fragment
 
+
 from xmodule.mako_module import MakoTemplateBlockBase
 from xmodule.progress import Progress
 from xmodule.seq_module import SequenceFields
 from xmodule.studio_editable import StudioEditableBlock
 from xmodule.x_module import STUDENT_VIEW, XModuleFields
 from xmodule.xml_module import XmlParserMixin
 
+from lms.djangoapps.completion.models import BlockCompletion
+
 log = logging.getLogger(__name__)
 
 # HACK: This shouldn't be hard-coded to two types
 # OBSOLETE: This obsoletes 'type'
 CLASS_PRIORITY = ['video', 'problem']
 
 
+def is_default_completable(block):
+    """
+    Returns True if the block can by completed by the default method.
```

Comment:
"the default method" is pretty vague - perhaps we should say "the default method (marking it as completed once it has been viewed)" or something ?